src/agent/include/process_options.hpp

```diff
@@ -10,7 +10,8 @@ void RestartAgent();
 void StartAgent();
 void StatusAgent();
 void StopAgent();
+void RestartAgent();
 void PrintHelp();
-void InstallService([[maybe_unused]] const std::string& exePath);
+void InstallService();
```

Comment:
Are these three functions necessary here?

Reply:
I think that this is the most straightforward approach to keep the PR focus on the development of service/daemon. We could improve this on a separate issue, an alternative would be to make the command line parser platform dependent as well, and only recognize certain options on specific platforms.

Reply:
Yes, we could improve this in the next one that completes the epic.